README.md

```diff
@@ -31,13 +31,15 @@ A very basic example of how to use this package is as follows:
 import dLuxToliman as dlT
 from matplotlib import pyplot as plt
 
-osys = dlT.TolimanOptics(psf_npixels=128, psf_oversample=1)  # creating Toliman optical system
+osys = dlT.TolimanOpticalSystem(psf_npixels=128)  # creating Toliman optical system
 source = dlT.AlphaCen(n_wavels=3, separation=8, position_angle=30)  # creating Alpha Centauri source
-psf = osys.model(source)  # running optical simulation
+tel = dlT.Toliman(source=source, osys=osys)  # creating Toliman telescope
+
+psf = tel.model()  # running optical simulation
 
 # plotting
 plt.imshow(psf ** .5, cmap='inferno')
-plt.title('Toliman $\sqrt{PSF}$')
+plt.title(r'Toliman $\sqrt{PSF}$')
```

Comment:
I personally think that `plt.title(r'Toliman $\sqrt{\text{PSF}}$')` looks better

Reply:
yeah it does but shot not regenerating the image

Reply:
That's Max!

Reply:
Maybe add this for future PRs